azure-pipelines-full-tests.yml

```diff
@@ -92,3 +72,99 @@ jobs:
         pythonVersion: '$(python.version)'
         profile: '2018-03-01-hybrid'
         fullTest: true
+
+- job: AutomationFullTestPython36ProfileLatest
+  displayName: Automation Full Test Python36 Profile Latest
+  timeoutInMinutes: 9999
+  strategy:
```

Comment:
Can we share the strategy among different python versions instead of duplication?

Reply:
Sorry, Matrix don't support `Cartesian product` style.